src/screens/AppletDetailsScreen.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC } from 'react';
+
+import { useTranslation } from 'react-i18next';
+
+import { AppletList } from '@app/entities/applet';
+import { AppletCalendar } from '@features/applet-calendar';
+import { Text, YStack, XStack, ScrollView, ImageBackground } from '@shared/ui';
+
+const AppletsDetailsScreen: FC = () => {
+  const { t } = useTranslation();
+
+  return (
+    <YStack jc={'flex-start'} flex={1}>
+      <ImageBackground>
+        <AppletCalendar />
+
+        <ScrollView>
+          <XStack mb="$3" py="$2.5" mx="$2" ai="center" bbc="$grey" bbw={1}>
+            <Text color="$darkerGrey">{t('additional:in_progress')}</Text>
+          </XStack>
+
+          <AppletList flex={1} px={14} mb={28} />
```

Comment:
There shouldn't be AppletList

Reply:
Please remove ScrollView and everything inside it. Alexey will pass there a widget